microsoft-edge/webview2/platforms/winui2-uwp.md

```diff
@@ -0,0 +1,229 @@
+
+
+<!-- ====================================================================== -->
+## Downloading files
+
+There are a couple of known limitations for current downloads behavior for WebView2 in UWP.
+
+
+<!-- ------------------------------ -->
+#### Save As
+
+Saving files via **Save As** is working and is enabled for WebView2 for UWP apps.  The files will be saved in the folder that the user selects.
+
+
+<!-- ------------------------------ -->
+#### Which folder the files are downloaded to
+
+If the host doesn't change the `ResultFilePath` of the downloaded file, the downloaded files will be downloaded to a subfolder with the app package's name in the `Downloads` folder.
+
+If the host changes the `ResultFilePath` of the downloaded file, the file will only be downloaded if the app has access to that file path by default. If you want to use a file location that the app doesn't have access to by default, you must set the corresponding capability. See [App capability declarations](/windows/uwp/packaging/app-capability-declarations) in the UWP documentation.
+
+
+<!-- ------------------------------ -->
+#### Downloads Hub
+
+Opening files and folders from the Downloads Hub is disabled.  Clicking on the file or folder icon won't open the respective file/folder.
+
+See also:
+* [Downloads](../concepts/overview-features-apis.md#downloads) in _Overview of WebView2 APIs_.
+
+
+<!-- ====================================================================== -->
+## XAML limitation
+
+XAML Island support requires additional work and may be considered for future releases.
+
+
+<!-- ====================================================================== -->
+## Setting DefaultBackgroundColor
+
+On WinUI 2, the `DefaultBackgroundColor` property is not exposed directly.  You can set the default background color by setting an environment variable, as follows:
+
+```csharp
+Environment.SetEnvironmentVariable("WEBVIEW2_DEFAULT_BACKGROUND_COLOR", "FF000000");
+```
+
+See also:
+* .NET: [WebView2.DefaultBackgroundColor Property](/dotnet/api/microsoft.web.webview2.winforms.webview2.defaultbackgroundcolor)
+* Win32: [ICoreWebView2Controller2::DefaultBackgroundColor property (get](/microsoft-edge/webview2/reference/win32/icorewebview2controller2#get_defaultbackgroundcolor), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2controller2#put_defaultbackgroundcolor)
+<!-- todo: relevant?
+* WinRT: [CoreWebView2Controller.DefaultBackgroundColor Property](https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2controller#defaultbackgroundcolor)
+-->
+<!--
+todo: relevant?
+* .NET: [CoreWebView2Controller.DefaultBackgroundColor Property](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2controller.defaultbackgroundcolor#microsoft-web-webview2-core-corewebview2controller-defaultbackgroundcolor)
```

Comment:
is this other .NET type member relevant?

.NET: CoreWebView2Controller.DefaultBackgroundColor Property
https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2controller.defaultbackgroundcolor#microsoft-web-webview2-core-corewebview2controller-defaultbackgroundcolor

the live article section that has a different .NET type member link:
https://learn.microsoft.com/microsoft-edge/webview2/get-started/winui2#setting-defaultbackgroundcolor

Reply:
closing as out of scope